source/includes/fundamentals/code-snippets/crud/bulk.rs

```diff
@@ -0,0 +1,199 @@
+use futures::{StreamExt, TryStreamExt};
+use mongodb::{
+    bson::doc,
+    bson::Document,
+    error::Result,
+    options::{CursorType, FindOptions},
+    Client, Collection,
+};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Mushroom {
+    name: String,
+    color: String,
+    edible: bool,
+}
+
+#[tokio::main]
+async fn main() -> mongodb::error::Result<()> {
+    let uri = "<connection string>";
+    let client = Client::with_uri_str(uri).await?;
+
+    let mushrooms: Collection<Document> = client.database("db").collection("mushrooms");
+
+    // begin-sample-data
+    let docs = vec![
+        Mushroom {
+            name: "portobello".to_string(),
+            color: "brown".to_string(),
+            edible: true,
+        },
+        Mushroom {
+            name: "chanterelle".to_string(),
+            color: "yellow".to_string(),
+            edible: true,
+        },
+        Mushroom {
+            name: "oyster".to_string(),
+            color: "white".to_string(),
+            edible: true,
+        },
+        Mushroom {
+            name: "fly agaric".to_string(),
+            color: "red".to_string(),
+            edible: false,
+        },
+    ];
+    // end-sample-data
+
+    let insert_many_result = mushrooms.insert_many(docs, None).await?;
+
+    // begin-insert
+    let mushrooms: Collection<Mushroom> = client.database("db").collection("mushrooms");
```

Comment:
S: reduce horizontal scroll (applies to all)
```suggestion
    let mushrooms: Collection<Mushroom> = client
        .database("db")
        .collection("mushrooms");
```